mapswipe_workers/mapswipe_workers/project_types/tile_map_service_grid/project.py

```diff
@@ -128,20 +151,21 @@ def validate_geometries(self):
         # for zoom level 18 this will be 5000 square kilometers
         # max zoom level is 22
         if self.zoomLevel > 22:
-            raise CustomError(f"zoom level is to large (max: 22): {self.zoomLevel}.")
+            raise CustomError(f"zoom level is too large (max: 22): {self.zoomLevel}.")
 
         max_area = (23 - int(self.zoomLevel)) * (23 - int(self.zoomLevel)) * 200
```

Comment:
This looks like black magic. Can you explain where the numbers come from in a comment?

Reply:
I added a better explanation for this.